File: POLSKI_SPOJ/C/PRIME_T.c
#include <stdio.h>
#include <stdbool.h>

bool isPrime(int primeNumber){
    int tmp = primeNumber / 2;
    if(primeNumber == 1) return false;
    if(primeNumber == 2 || primeNumber == 3) return true;
    else {
        for(int i = 2; i <= tmp; i++){
            if(primeNumber % i == 0) return false;
        }
    }
    return true;
}

int main() {
    int num;
    int n;
    scanf("%d", &n);
    if(n >= 0 && n <= 100000) {
        for(int i = 0; scanf("%d", &num) && num <= 10000 && num > 0) {
            if (isPrime(num) == true) printf("TAK\n");
            else if (isPrime(num) == false) printf("NIE\n");
        }
    }
    return 0;
}

File: POLSKI_SPOJ/Python/PRIME_T.py
def isPrime(num):
    if num == 1: return False
    for x in range(2, int(num ** 0.5 + 1)):
        if num % x == 0: return False
    return True


numberOfCases = input()
for x in range(int(numberOfCases)):
    number = int(input())
    print("NIE") if isPrime(number) is False else print("TAK")
